RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -200,6 +200,27 @@ - (bool) isTablet
   return [[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomPad;
 }
 
+- (NSNumber*) fontScale
```

Comment:
Please add a comment linking to https://developer.apple.com/ios/human-interface-guidelines/visual-design/typography/